pkg/flowkit/config/loader.go

```diff
@@ -130,12 +130,15 @@ func (l *Loader) Load(paths []string) (*Config, error) {
 	// special case for default configs
 	// try to load local config and only if not found try to load global config
 	if IsDefaultPath(paths) {
-		conf, err := l.LoadConfig(DefaultPath)
-		if err == nil {
+		conf, err := l.loadConfig(DefaultPath)
+		if err == nil { // if we could load it then process it
 			return l.postprocess(conf)
 		}
+		if err != ErrDoesNotExist {
```

Comment:
Consider replacing with
```suggestion
		if errors.Is(err, ErrDoesNotExist) {
```

Reply:
Great catch, thank you (not equal tho)